provider/core/src/lib.rs

```diff
@@ -174,10 +174,14 @@ pub mod prelude {
     pub use crate::any::AsDynProviderAnyMarkerWrap;
     #[cfg(feature = "serde")]
     pub use crate::serde::AsDeserializingBufferProvider;
+
+    pub use crate::yoke;
+    pub use crate::zerofrom;
 }
 
-/// Re-export of the yoke crate for convenience of downstream implementors.
+/// Re-export of the yoke and zerofrom crates for convenience of downstream implementors.
 pub use yoke;
+pub use zerofrom;
```

Comment:
This is weird, you're re-exporting `yoke` and `zerofrom` at the root, then re-re-exporting them in `prelude`, and then re-re-re-exporting them with the whole `prelude` in the root.

```suggestion
    pub use yoke;
    pub use zerofrom;
}
```

Reply:
I fixed up the reexport to be cleaner though.